sql/pre_install/tables.sql

```diff
@@ -367,6 +367,45 @@ CREATE INDEX IF NOT EXISTS remote_txn_data_node_name_idx ON _timescaledb_catalog
 
 SELECT pg_catalog.pg_extension_config_dump('_timescaledb_catalog.remote_txn', '');
 
+-- This table stores information about the stage that has been completed of a
+-- chunk move/copy activity (cma)
+--
+-- Cleanup activity can query and check if the backend is running. If the
+-- backend has exited then we can commence cleanup. The cleanup
+-- activity can also do a diff with the "time_start" value to ascertain if
+-- the entire end-to-end activity is going on for too long
+--
+-- We also track the end time of every stage. A diff with the current time
+-- will give us an idea about how long the current stage has been running
+--
+-- Entry for a chunk move/copy activity gets deleted on successful completion
+--
+-- We don't want to pg_dump this table's contents
+--
+-- TODO: check the locking requirements
```

Comment:
I guess those TODO comments are leftovers

Reply:
These are actual TODOs that are remaining. We have figured out the `locking` requirements yet. Nor have to figured out how to check if the activity has completed or aborted.